clients/unshaded/src/main/java/tachyon/client/lineage/LineageMasterClient.java

```diff
@@ -88,8 +87,8 @@ public synchronized long createLineage(List<TachyonURI> inputFiles, List<Tachyon
       try {
         return mClient.createLineage(inputFileStrings, outputFileStrings,
             job.generateCommandLineJobInfo());
-      } catch (FileDoesNotExistException e) {
-        throw e;
+      } catch (TachyonTException e) {
+        throw new TachyonException(e);
```

Comment:
it seems on the fooClient.java, we simply convert `TachyonTException` to `TachyonException`, and rely on the upper level to do correct unwrap, right?